components/lib/dialog/Dialog.js

```diff
@@ -10,6 +10,7 @@ import { Portal } from '../portal/Portal';
 import { Ripple } from '../ripple/Ripple';
 import { DomHandler, IconUtils, ObjectUtils, UniqueComponentId, ZIndexUtils, mergeProps } from '../utils/Utils';
 import { DialogBase } from './DialogBase';
+import { useOnEscape } from '../../lib/hooks/Hooks';
```

Comment:
Lets rename it `useOnEscapeKey` as its more specific the word "escape" can have many meanings.

Reply:
Done @melloware